x-pack/plugins/security_solution/public/flyout/right/components/host_entity_overview.tsx

```diff
@@ -206,29 +212,33 @@ export const HostEntityOverview: React.FC<HostEntityOverviewProps> = ({ hostName
           </EuiFlexItem>
         </EuiFlexGroup>
       </EuiFlexItem>
-      <EuiFlexItem>
-        <EuiFlexGroup>
-          <EuiFlexItem>
-            <OverviewDescriptionList
-              dataTestSubj={ENTITIES_HOST_OVERVIEW_OS_FAMILY_TEST_ID}
-              descriptionList={hostOSFamily}
-            />
-          </EuiFlexItem>
-          <EuiFlexItem>
-            {isAuthorized ? (
-              <DescriptionListStyled
-                data-test-subj={ENTITIES_HOST_OVERVIEW_RISK_LEVEL_TEST_ID}
-                listItems={[hostRiskLevel]}
-              />
-            ) : (
+      {isRiskScoreLoading || isHostDetailsLoading ? (
+        <FlyoutLoading data-test-subj={ENTITIES_HOST_OVERVIEW_LOADING_TEST_ID} />
```

Comment:
I think this section and the mirror section under `user_entity_overview` could also be good candidates for `EuiSkeletonText`, wdyt?